### DISCUSSION TITLE
Get Em

### DISCUSSION THREAD

**Discussion by LouderthanThunder** [Lambosaurus/hives-and-torches#37] [Feats]
Each creature gets an additional damage for each additional creature targeting the same opponent. 

Makes horde minions a threat

**Comment by Lambosaurus** [comment]
I think horde minions will already be a threat due to our consistent damage and action economy.

**Reply by Lambosaurus** [reply]
Even so, perhaps we need a new category for monster options?

**Reply by LouderthanThunder** [reply]
Yeah

**Reply by Lambosaurus** [reply]
Actually, I have a feeling that this is just duplicate of pack tactics.
